src/components/Lane/LaneHeader.js

```diff
@@ -6,12 +6,12 @@ import {Title, LaneHeader, RightContent } from 'styles/Base'
 import LaneMenu from './LaneHeader/LaneMenu'
 
 const LaneHeaderComponent = ({
-  updateTitle, canAddLanes, onDelete, onDoubleClick, inlineEditTitle, label, title, titleStyle, labelStyle, t
+  updateTitle, canAddLanes, onDelete, onDoubleClick, inlineEditTitle, label, title, titleStyle, labelStyle, t, draggable, laneDraggable
```

Comment:
I think it's enough to pass `laneDraggable` property and set it to `true` when calling LaneHeader, here - src/styles/Base.js 

Because it is not good to know a lot about up level components and properties.